### PR TITLE
everything i have done so far

### PR DESCRIPTION
**Pull request by Ronabul**
hello please give me a CR

### CODE REVIEW

File: guessing_game/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    let mut guess;
+
+    loop {
+        println!("Please enter your input:");
+
+        guess = String::new();
+
+        io::stdin()
+            .read_line(&mut guess)
+            .expect("Failed to read line!");
```
Comment (arhimenez) [inline guessing_game/src/main.rs]: bishbash

File: short_fibonacchi/src/main.rs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+    return buff;
+}
+
+
+fn main() {
+    let buff = fib(15);
+
+    for i in buff{
+        println!("{i}")
```
Comment (arhimenez) [inline short_fibonacchi/src/main.rs:26]: @Ronabul

**Review by arhimenez — CHANGES_REQUESTED** [review]
do pls :) @Ronabul

File: guessing_game/src/main.rs (hunk 1 above)
Comment (Ronabul) [reply guessing_game/src/main.rs]: i have changed this in a new commit please take a look

File: short_fibonacchi/src/main.rs (hunk 2 above)
Comment (Ronabul) [reply short_fibonacchi/src/main.rs:26]: i do not agree with you, this is the correct way of doing it.

**Review by arhimenez — APPROVED** [review]
nice hah:)

**Pull request merged**